spec/timezone.html

```diff
@@ -420,9 +420,9 @@ <h1>
         1. Assert: ! IsValidEpochNanoseconds(ℤ(_epochNanoseconds_)) is *true*.
         1. Let _remainderNs_ be _epochNanoseconds_ modulo 10<sup>6</sup>.
         1. Let _epochMilliseconds_ be 𝔽((_epochNanoseconds_ - _remainderNs_) / 10<sup>6</sup>).
-        1. Let _year_ be ℝ(! YearFromTime(_epochMilliseconds_)).
-        1. Let _month_ be ℝ(! MonthFromTime(_epochMilliseconds_)) + 1.
-        1. Let _day_ be ℝ(! DateFromTime(_epochMilliseconds_)).
+        1. Let _year_ be ! EpochTimeToEpochYear(_epochMilliseconds_).
+        1. Let _month_ be ! EpochTimeToMonthInYear(_epochMilliseconds_) + 1.
+        1. Let _day_ be ! EpochTimeToDate(_epochMilliseconds_).
```

Comment:
```suggestion
        1. Let _year_ be EpochTimeToEpochYear(_epochMilliseconds_).
        1. Let _month_ be EpochTimeToMonthInYear(_epochMilliseconds_) + 1.
        1. Let _day_ be EpochTimeToDate(_epochMilliseconds_).
```
If these are equations, they can't return a completion record, so we don't need to assert normal completion with `!`.